### ISSUE TITLE
Error in userIdTargeting.js: TypeError: "Object(...)(...).getUserIds is not a function

### ISSUE THREAD

**Issue by musikele** [prebid/Prebid.js#5072]
## Type of issue
When integrating module userIdTargeting.js, we see this exception in the logs: 
```
ERROR: Error executing handler: events.js TypeError: "Object(...)(...).getUserIds is not a function"
```

## Description

UserIdTargeting module is executing these lines: 

```
export function init(config) {
  events.on(CONSTANTS.EVENTS.AUCTION_END, function() {
    userIdTargeting((getGlobal()).getUserIds(), config.getConfig(MODULE_NAME)); <-- here's the error 
  })
}
```

If the returned object by`getGlobal()` does not have the method `getUserIds()`, we see the exception above.

## Steps to reproduce

- latest commit on branch master 
- npx gulp serve 
- modify hello_world.html to show debug logs 
- connect to http://localhost:9999/integrationExamples/gpt/hello_world.html
- check the logs 

## Test page

check steps above. 

### Expected results

no error in logs. 

### Actual results
```
 Prebid ERROR: Error executing handler: events.js TypeError: "Object(...)(...).getUserIds is not a function"
    init userIdTargeting.js:57
    _dispatch events.js:67
    _each utils.js:441
    _dispatch events.js:64
    emit events.js:98
    executeCallback auction.js:170
    ....
```

## Platform details
I see the error on: 
- latest commit on master: 0d4527af 
- latest tag: 3.14.0

**Comment by pm-harshad-mane** [comment]
Hello @musikele ,
Have you included the userId module in PrebidJS while building?

**Comment by musikele** [comment]
@pm-harshad-mane, I run prebid with all modules built in (e.g. `npx gulp serve --notest`), so I think the userId module is included. 

Anyway, I've just tried to run 
```
npx gulp serve --modules userId,rubiconBidAdapter,userIdTargeting --notest
```
and I see the same error in console.

**Comment by pm-harshad-mane** [comment]
@musikele , i pulled the latest code, and ran http://localhost:9999/integrationExamples/gpt/hello_world.html?pbjs_debug=true I do not see the mentioned error. I see this log ``` Prebid INFO: userIdTargeting: Invalid config found, not sharing userIds externally. ```

**Comment by musikele** [comment]
It looks that when the hostname is localhost the error doesn't show up. Try to set in `etc/hosts` an alias, like `127.0.0.1  customwebsite.com`. 
Also, the error shows up in Firefox, not in Chrome.

**Comment by pm-harshad-mane** [comment]
Thanks @musikele , I will check.

**Comment by pm-harshad-mane** [comment]
@musikele I checked again in FF with a host file entry, no errors are observed.

**Comment by musikele** [comment]
@pm-harshad-mane
I did more tests anyway. I cannot understand why I am the only one seeing this bug :)
Here's what I tested: 
- chrome, no errors
- safari, no errors, 
- firefox 75, no errors 
- firefox Developer Edition (version 76), I see the error

I use Firefox Developer Edition as my default browser that's why I see this error. Usually Firefox Dev edition is one major version forward respect to the regular firefox, so at the next release this bug may pop up in regular firefox too (unless firefox devs will change something in the meantime). 

if this bug only happens to me, and you're unable to replicate, feel free to close it.

**Comment by smenzer** [comment]
Any updates here? Is this issue still happening for you @musikele ?

**Comment by musikele** [comment]
No, I don't see this issue anymore, neither on firefox developer edition. Going to close this one
